fec_eregs/docs/Team Practices.md

```diff
@@ -0,0 +1,143 @@
+# Team Practices
+
+This page is meant to describe the team practices that the FEC Legal Resources team operate under.  
+
+It is a living document that will be amended as the team learns what works sprint by sprint.
+
+Many of these practices come from Agile development processes; with no specific framework being adopted.
+
+# Team Members 
+(once in GitHub link it to 18F .yml)
+* Aaron Borden - Front End Developer
+* Annalee Flower Horne - Back End Developer
+* Anthony Garvan - Back End Developer
+* Ed Mullen - UX and Design
+* Leah Bannon - Engagement Manager
+* Porta Antiporta - Product Steward
+* Tadgh O'Higgins - Back End Developer
+
+# Tools & Repositories
+WIP - integration between tools is still under investigation.  Tools were selected to give different views of our project
+
+## Story Map - [Storiesonboard](https://18f.storiesonboard.com/m/fec)
+Story maps will serve as the vision/roadmap for our product.  It will describe the different users and activities/goals they want to achieve as they interact with beta.fec.gov
+## Sprint Taskboard - [Trello](https://trello.com/b/1NNW09CL/fec-eregs-discovery)
+Trello will be our used as our taskboard.  All work the team is doing should be reflected on the taskboard.  Backlog grooming/planning/estimation/demos will be driven by the task board.  
+
+Owners of each card should comment frequently to provide visibility to progress (or lack thereof).  
+
+While the product steward will likely be creating the bulk of the cards (in collaboration with the FEC product owner), any team member can create cards that will be prioritized into grooming sessions.
+
+TBD - Creating / Syncing cards from whatever forum will be used for user feedback, reported defects etc.
+
+## GitHub
+Our code repositories.  
+The following are GH repositories of note:
+* [fec-eregs](https://github.com/18F/fec-eregs)
+* [regulations-core](https://github.com/18F/regulations-core)
+* [regulations-site](https://github.com/18F/regulations-site)
+* [regulations-parser](https://github.com/18F/regulations-parser)
+* [openFEC-web-app](https://github.com/18F/openFEC-web-app)
+* [fec-styles](https://github.com/18F/fec-styles)
+* [fec-cms](https://github.com/18F/fec-cms)
+
+TBD - GH Issues syncing wih Trello
```

Comment:
Zapier - don't have auth to use.
IFTTT - tried, but the triggers are too basic for meaningful syncing.
Trello Business Class - unsure if/when we'd get access.